### DISCUSSION TITLE
Dowloads bar hangs when dowloading model

### DISCUSSION THREAD

**Discussion by xlf425** [open-webui/open-webui#3098] [General]
# Bug Report

## Description

**Bug Summary:**
Have to refresh several times wep app while downloading a models. While refreshing, I have to enter model name again and the status bar shows the download correct percent amount and resume at the correct state.

**Steps to Reproduce:**
-dowload Ollama webUI with GPU support with the Docker command provided in the ReadMe

**Expected Behavior:**
Downloads bar evolute normally

**Actual Behavior:**
see Bug Summary

## Environment

- **Open WebUI Version:** [v0.3.3]
- **Ollama (if applicable):** [0.1.41]

- **Operating System:** [Windows 11]
- **Browser (if applicable):** [Brave]

## Reproduction Details

**Confirmation:**

- [x ] I have read and followed all the instructions provided in the README.md.
- [x ] I am on the latest version of both Open WebUI and Ollama.
- [x ] I have included the browser console logs.
- [x ] I have included the Docker container logs.

## Logs and Screenshots
![Capture d'écran 2024-06-12 204846](https://github.com/open-webui/open-webui/assets/44872721/14dfd130-e4ea-4a7e-9c77-ddd5d8d49391)

**Browser Console Logs:**
[Include relevant browser console logs, if applicable]

**Docker Container Logs:**
[Include relevant Docker container logs, if applicable]

**Screenshots (if applicable):**
[Attach any relevant screenshots to help illustrate the issue]

## Installation Method

[Describe the method you used to install the project, e.g., manual installation, Docker, package manager, etc.]

## Additional Information

[Include any additional details that may help in understanding and reproducing the issue. This could include specific configurations, error messages, or anything else relevant to the bug.]

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!

**Comment by silentoplayz** [comment]
> ## Environment
> Open WebUI Version: [e.g., 0.1.120]
> Ollama (if applicable): [e.g., 0.1.30, 0.1.32-rc1]
> Operating System: [Windows 11]
> Browser (if applicable): [Brave]

It appears that both your Open WebUI and Ollama installations are outdated. This may be causing the issue at hand. Please update them accordingly to their latest versions and let us know if this issue still occurs afterwards!

**Open WebUI** - v0.3.4 [(latest)](https://github.com/open-webui/open-webui/releases/tag/v0.3.4)
**Ollama** - v0.1.43 [(latest)](https://github.com/ollama/ollama/releases/tag/v0.1.43)

**Comment by xlf425** [comment]
My bad, here's the update. Issue was edited

- **Open WebUI Version:** [v0.3.3]
- **Ollama (if applicable):** [0.1.41]

**Reply by silentoplayz** [reply]
Does the issue still appear to happen on the latest version of Open WebUI?

**Reply by Sercurio** [reply]
got exactly the same problem on 
MacOS: Sonoma 14.5

![Uploading Capture d’écran 2024-06-13 à 09.31.42.png…]()

**Comment by silentoplayz** [comment]
Thank you for making us aware of this issue. To assist you more effectively, I would appreciate it if you could re-open the issue.
